packages/core/admin/package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "prepublishOnly": "node ./scripts/build.js",
-    "test": "echo \"no tests yet\"",
+    "test:unit": "jest --verbose",
+    "test:front": "cross-env IS_EE=true jest --config ./jest.config.front.js",
```

Comment:
It would be great if we could have both a watch command and a command to run the tests in ce mode.